lookout/style/typos/templates/scores.md.jinja2

```diff
@@ -0,0 +1,19 @@
+### Typos correction report
+
+| Score type | Accuracy | Precision | Recall | F1-score |
+|-----------:|:---------|:----------|:-------|:---------|
+{% for mode in scores %}
+    | `{{ mode }}` |
+    {% for metric in ["accuracy", "precision", "recall", "f1"] %}
+    {{ "%.3f" % scores[mode]["accuracy"] -}} |
```

Comment:
```suggestion
    {{ "%.3f" % scores[mode][metric] -}} |
```
?

Can I also ask you to align nimbers with spases to make raw table looks prety?
I mean
```
| Score type | Accuracy | Precision | Recall | F1-score |
|-----------:|:---------|:----------|:-------|:---------|
| type | 0.123 | 0.123 | 0.123 | 0.123 |
```
to 
```
| Score type | Accuracy | Precision | Recall | F1-score |
|-----------:|:---------|:----------|:-------|:---------|
|       type |    0.123 |     0.123 |  0.123 |    0.123 |
```